documentation/haproxy_install.md

```diff
@@ -17,38 +17,39 @@ Introduced: v4.0.0
 
 This resource also uses the following partial resources:
 
-* [_config_file](https://github.com/sous-chefs/haproxy/tree/master/documentation/partial_config_file.md)
+* [\_config_file](https://github.com/sous-chefs/haproxy/tree/master/documentation/partial_config_file.md)
```

Comment:
Typo?

Reply:
Overzealous application of markdown lint, it looks like :)